modulemd/v2/modulemd-defaults-v1.c

```diff
@@ -46,6 +47,105 @@ G_DEFINE_TYPE (ModulemdDefaultsV1,
                MODULEMD_TYPE_DEFAULTS)
 
 
+static gboolean
+modulemd_defaults_v1_equals (ModulemdDefaults *self_1, ModulemdDefaults *self_2, GError **error)
```

Comment:
This function doesn't need a GError return. Either they are identical or they are not, but it shouldn't be possible for any of the comparisons to actually fail (barring programming errors like dereferencing NULL). In the unlikely event that an error is detected, it should just return FALSE.